sci-electronics/kicad-packages3d/kicad-packages3d-5.0.0-r1.ebuild

```diff
@@ -12,10 +12,12 @@ SRC_URI="https://github.com/KiCad/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
 LICENSE="CC-BY-SA-4.0"
 SLOT="0"
 KEYWORDS="~amd64"
-IUSE=""
+IUSE="occ oce"
+
+REQUIRED_USE="?? ( occ oce )"
```

Comment:
Thanks för reviewing. I'm going to update the ebuild the day after tomorrow, when I'm back home.
I first wanted to use the at least one operator, when I noticed kicked using at most one and changed it to reflect the same usage. I agree in using at least one operator being a better choice.

Reply:
Sorry @zpuskas, I somehow missed to review this after christmas and forgot about it. I updated it and will push it, but I'm not sure whether we need this update any longer, as 5.0.1 has come out since.